client/src/components/advancedSearch/TaskFilter.tsx

```diff
@@ -138,7 +323,7 @@ const TaskFilter = ({
   }
 }
 
-export const TaskMultiFilter = ({ ...props }) => <TaskFilter {...props} multi />
+export const TaskMultiFilter = ({ ...props }) => <TaskFilter {...props} />
```

Comment:
Remove this export and use the default export everywhere, it is always a multi-filter. Also remove the unused `deserialize` and rename `deserializeMulti` to `deserialize`.